conf/modules.config

```diff
@@ -18,23 +55,351 @@ process {
+                params.wfmash_block_length     ? "-l ${params.wfmash_block_length}"  : "-l ${params.wfmash_segment_length * 5}",
+                "-k ${params.wfmash_mash_kmer}",
+                "-H ${params.wfmash_mash_kmer_thres}",
+                "${wfmash_sparse_map_cmd}",
+                params.wfmash_temp_dir         ? "-B ${wfmash_temp_dir}"             : "",
+                "-m",
+            ].join(" ").trim()
+        }
         publishDir = [
-            path: { "${params.outdir}/pipeline_info" },
+            path: { "${params.outdir}/wfmash_map" },
             mode: params.publish_dir_mode,
-            pattern: '*_versions.yml'
+            pattern: '*.paf'
+        ]
+    }
+
+    withName: SPLIT_APPROX_MAPPINGS_IN_CHUNKS {
+        ext.args = {
+            [
+                "${params.wfmash_chunks}",
+            ].join(" ").trim()
+        }
+        publishDir = [
+            path: { "${params.outdir}/split_approx_mappings_in_chunks" },
+            mode: params.publish_dir_mode,
+            pattern: '{*.paf}'
+        ]
+    }
+
+    withName: WFMASH_ALIGN {
+        ext.args = {
+            [
+                "-n ${params.n_haplotypes - 1}",
+                "-s ${params.wfmash_segment_length}",
+                "-p ${params.wfmash_map_pct_id}",
+                params.wfmash_merge_segments   ? "-M"                                : "",
+                params.wfmash_exclude_delim    ? "-Y ${params.wfmash_exclude_delim}" : "-X",
+                params.wfmash_no_splits        ? "-N"                                : "",
+                params.wfmash_block_length     ? "-l ${params.wfmash_block_length}"  : "-l ${params.wfmash_segment_length * 5}",
+                "-k ${params.wfmash_mash_kmer}",
+                "-H ${params.wfmash_mash_kmer_thres}",
+                "${wfmash_sparse_map_cmd}",
+                params.wfmash_temp_dir         ? "-B ${wfmash_temp_dir}"             : "",
+                "--invert-filtering",
+            ].join(" ").trim()
+        }
+        publishDir = [
+            path: { "${params.outdir}/wfmash_align" },
+            mode: params.publish_dir_mode,
+            pattern: '*.paf'
+        ]
+    }
+
+    withName: SEQWISH {
+        ext.prefix = {"${meta.id}.seqwish"}
+        ext.args = {
+            [
+                "-k ${params.seqwish_min_match_length}",
+                "-f ${params.seqwish_sparse_factor}",
+                "-B ${params.seqwish_transclose_batch}",
+                "-P",
+                params.seqwish_temp_dir        ? "--temp-dir ${params.seqwish_temp_dir}" : ""
+            ].join(" ").trim()
+        }
+        publishDir = [
+            path: { "${params.outdir}/seqwish" },
+            mode: params.publish_dir_mode,
+            pattern: '*.gfa'
+        ]
+    }
+
+    withName: SMOOTHXG {
+        ext.args = {
+            [
+                "-T ${task.cpus}", // TODO make this accessible using a parameter
```

Comment:
No, because this sets the number of threads for the POA phase of the module.
In the module itself there already is `-t ${task.cpus}` https://github.com/nf-core/modules/blob/c4cc1db284faba9fc4896f64bddf7703cedc7430/modules/nf-core/smoothxg/main.nf#L26.

Reply:
Am I making sense to you @FriederikeHanssen ?